conn.go

```diff
@@ -31,24 +32,24 @@ type Stats struct {
 
 // NewConn establishes a connection to the DB. After obtaining the connection,
 // it sends "SET NAMES utf8" command to the DB
-func NewConn(username, password, protocol, address, database string) (*Conn, error) {
-	return NewConnContext(context.Background(), username, password, protocol, address, database)
+func NewConn(username, password, protocol, address, database string, readTimeout time.Duration) (*Conn, error) {
+	return NewConnContext(context.Background(), username, password, protocol, address, database, readTimeout)
 }
 
 // NewConnContext establishes a connection to the DB. After obtaining the connection,
 // it sends "SET NAMES utf8" command to the DB
 //
 // Go Context is only used to establish a TCP connection.
 // TODO use Go Context to establish a MySQL connection.
-func NewConnContext(ctx context.Context, username, password, protocol, address, database string) (*Conn, error) {
+func NewConnContext(ctx context.Context, username, password, protocol, address, database string, readTimeout time.Duration) (*Conn, error) {
```

Comment:
Would be more pretty to split line or introduce structure to combine arguments?

Reply:
I split it into multiple lines.